.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Report a bug.
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+\-------------------------------------------------------------------------------------------------
+**IS THIS BUG RELATED TO EXTERNAL TOOLS? (e.g. git, less, etc.)**
+
+If it is, there's a good chance we might need to know a little bit more about the environment
+which you're running `bat` on.
+
+**If you're on Windows:**  
+Please tell us about your:
```

Comment:
Maybe: "In the last section of this template, please tell us about …".

Reply:
I changed the header to say "Please tell us about the following at the bottom of the bug report", which should cover that.